### DISCUSSION TITLE
Does proxy URL support the existing host?

### DISCUSSION THREAD

**Discussion by KevinSheeranxyj** [mockoon/mockoon#1484] [General]

**Comment by 255kb** [comment]
This is the purpose of the proxy mode. It redirects all the calls to non-existing endpoints to another URL, any URL you want. 
I suggest you follow this tutorial to learn more: https://mockoon.com/tutorials/partial-mocking-proxy/
You can also have a look at the docs: https://mockoon.com/docs/latest/server-configuration/proxy-mode/
